install_config/install/docker_registry.adoc

```diff
@@ -84,6 +86,24 @@ $ oc volume deploymentconfigs/docker-registry \
      --source='{"nfs": { "server": "<fqdn>", "path": "/path/to/export"}}'
 ----
 
+There are known issues when using multiple registry replicas with the same NFS
+volume. Due to a client side caching, file system changes are propagated with
```

Comment:
I'd replace from "Due to..." to the end of the paragraph with something like "Factors such as NFS client side caching, network latency, and layer size can all contribute to potential errors that might occur when pushing an image using the default round-robin load balancing configuration. One possible workaround is to set the `docker-registry` service's `sessionAffinity` to `ClientIP`:"